testsuite/tests/regression/pr6608/Makefile

```diff
@@ -0,0 +1,32 @@
+#**************************************************************************
+#*                                                                        *
+#*                                OCaml                                   *
+#*                                                                        *
+#*                     Thomas Refis, Jane Street Europe                   *
```

Comment:
No need for a copyright header in a testsuite Makefile (apparently forgotten where this one came from!)